reorder_python_imports.py

```diff
@@ -651,7 +672,10 @@ def _is_py3(args):  # type: (argparse.Namespace) -> bool
 
 def _six_replaces(args):  # type: (argparse.Namespace) -> List[ImportToReplace]
     if _is_py3(args):
-        return [_validate_replace_import(s) for s in SIX_RENAMES]
+        return [
+            _validate_replace_import(s)
+            for s in itertools.chain(MOCK_RENAMES, SIX_RENAMES)
```

Comment:
I would just use `+` instead of itertools probably

Reply:
sure